tests/xbot/common/command/scripted/TestCommandFactory.java

```diff
@@ -0,0 +1,45 @@
+package xbot.common.command.scripted;
+
+import org.junit.Ignore;
+
+import edu.wpi.first.wpilibj.command.Command;
+import xbot.common.command.scripted.ScriptedCommandFactory;
+import xbot.common.command.scripted.ScriptedCommandProvider;
+
+@Ignore
+public class TestCommandFactory implements ScriptedCommandFactory {
+    private ExecutionCounterCommandProvider lastExecutionCounterCommandProvider;
+    
+    @Override
+    public ScriptedCommandProvider getProviderForName(String commandTypeName) {
+        if(commandTypeName.equals("CounterCommand"))
```

Comment:
Are we going to have to add an entry here for every command to be used in the script?